src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  static die() {
+    Animal.alive = Animal.alive.filter(elememt => elememt.health > 0);
+  }
```

Comment:
use this as just filter in bite() function. It's not logically a little, we don't have method `die` in real life :))